apps/admin/frontend/src/app_root.tsx

```diff
@@ -142,16 +147,21 @@ export function AppRoot({
   );
 
   const saveElection: SaveElection = useCallback(
-    async (electionJson) => {
-      await store.configure(electionJson);
+    async (electionData) => {
+      return new Promise((resolve) => {
+        configureMutation.mutate(
+          {
+            electionData,
+          },
+          {
+            onSuccess: resolve,
+          }
+        );
+      });
     },
-    [store]
+    [configureMutation]
   );
```

Comment:
Would like to move this out of top-level `Context`, but the definition screen is using it also so I left it here. Could move it down anyway.

@mattroe can we axe the definition editing screens on VxAdmin? It is work to maintain them.

Reply:
I probably won't axe them in this PR because it has other implications, so I'll just move it down.